content/en/docs/instrumentation/java/manual.md

```diff
@@ -233,47 +250,40 @@ artifact coordinates, see [releases]. For semantic convention releases, see
 
 {{% /tab %}} {{% /tabpane %}}
 
+Throughout this documentation you will add dependencies. For a full list of
+artifact coordinates, see [releases]. For semantic convention releases, see
+[semantic-conventions-java].
+
+[releases]: https://github.com/open-telemetry/opentelemetry-java#releases
+[semantic-conventions-java]:
+  https://github.com/open-telemetry/semantic-conventions-java/releases
+
 ### Initialize the SDK
 
 {{% alert title="Note" color="info" %}} If you’re instrumenting a library,
 **skip this step**. {{% /alert %}}
 
-If you instrument a Java app, install the dependencies for the OpenTelemetry
-SDK.
+If you instrument a Java app, install the dependencies for the OpenTelemetry SDK
+in addition to the dependencies for the OpenTelemetry API.
```

Comment:
The phrasing here is odd. Your change is fine but I wonder if we should correct it. 

> If you instrument a Java app, install the dependencies for the OpenTelemetry SDK

That's a really poor introduction to the SDK. Maybe something like:

"The OpenTelemetry API provides a set of interfaces for collecting telemetry, but the data is dropped without an implementation. The OpenTelemetry SDK is the implementation of the OpenTelemetry API provided by OpenTelemetry. To use it, begin by installing dependencies:"